packages/android_alarm_manager_plus/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 1.0.1
+
+ - **REFACTOR**: rename e2e test files to _plus_e2e to be compatible with melos (#40).
```

Comment:
I think we don't really need a new release for this

Reply:
I absolutely agree and I have no idea how this came to my PR! I have not changed this at all!